package.json

```diff
@@ -7,7 +7,7 @@
   "name": "sass",
   "devDependencies": {
     "node-sass": "^4.11.0",
-    "chokidar": "^2.0.0",
+    "chokidar": "^3.3.0",
```

Comment:
I'd like this to match the distributed `package.json`'s constraint, which is `>=2.0.0 <4.0.0`.